src/main/java/duckie/Duckie.java

```diff
@@ -0,0 +1,65 @@
+package duckie;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import duckie.command.Command;
+import duckie.exception.DuckieException;
+import duckie.task.TaskList;
+
+/**
+ * Main file for the chatbot Duckie
+ */
+public class Duckie {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Instantiate the Duckie object together with the filePath of duckie file
+     * @param filePath Duckie file location
+     */
+    public Duckie(String filePath) {
+        ui = new Ui();
+        ui.showIntro();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DuckieException e) {
+            ui.showError(e.getMessage());
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Activates the chatbot to take in commands
+     */
+    public void run() {
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                String fullCommand = ui.readCommand();
+                ui.showLine();
+                Command c = Parser.parse(fullCommand);
+                c.execute(tasks, ui, storage);
+                isExit = c.isExit();
+            } catch (DuckieException e) {
+                ui.showError(e.getMessage());
+            } finally {
+                ui.showLine();
+            }
+        }
+    }
+
+    /**
+     * Main method activating run() method
+     * @param args
```

Comment:
Could consider removing javadocs for functions like public static void main which are ubiquitous in Java.